common/src/main/java/com/tencent/rss/common/config/ConfigUtils.java

```diff
@@ -178,12 +178,12 @@ static Double convertToDouble(Object o) {
     return configOptionList;
   }
 
-  public static Function<Long, Boolean> positiveLongValidator = value -> value > 0;
+  public static Function<Long, Boolean> positiveLongValidator = value -> value > 0L;
 
-  public static Function<Long, Boolean> non_negativeLongValidator = value -> value >= 0;
+  public static Function<Long, Boolean> non_negativeLongValidator = value -> value >= 0L;
 
-  public static Function<Long, Boolean> positiveIntegerValidator =
-      value -> value > 0L && value <= Integer.MAX_VALUE;
+  public static Function<Integer, Boolean> positiveIntegerValidator =
```

Comment:
This type should be Long.

Reply:
Yes, using Integer will always return true for the check `value <= Integer.MAX_VALUE` but we can not apply i a Integer type config option. I will revert this change and use `positiveIntegerValidator2` since I want to use int type config option.